src/nitin/array/Array.java

```diff
@@ -0,0 +1,57 @@
+package nitin.array;
+
+public class Array {
+	
+	int getMinNumber(int num[]) {
+		int min=num[0];
+		for(int index=0;index<num.length;index++) {
```

Comment:
Video sent on skype for this code, pls go through it

Also Check this code for reference
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/318/files